sdk/monitor/azquery/custom_client.go

```diff
@@ -109,47 +111,82 @@ func (e *ErrorInfo) Error() string {
 // Row of data in a table, types of data used by service specified in LogsColumnType
 type Row []any
 
-// Table - Contains the columns and rows for one table in a query response.
-type Table struct {
-	// REQUIRED; The list of columns in this table.
-	Columns []*Column `json:"columns,omitempty"`
+// TimeInterval specifies the time range over which to query.
+// Use NewTimeInterval() for help formatting.
+// Follows the ISO8601 time interval standard with most common format being startISOTime/endISOTime.
+// Use UTC for all times.
+type TimeInterval string
 
-	// REQUIRED; The name of the table.
-	Name *string `json:"name,omitempty"`
+// NewTimeInterval creates a ISO8601TimeInterval for use in a query.
+// Use UTC for start and end times.
+func NewTimeInterval(start time.Time, end time.Time) TimeInterval {
+	return TimeInterval(start.Format(time.RFC3339) + "/" + end.Format(time.RFC3339))
+}
 
-	// REQUIRED; The resulting rows from this query.
-	Rows []Row `json:"rows,omitempty"`
+// Times returns the interval's start and end times if it's in the format startISOTime/endISOTime, else it will return an error.
+func (i TimeInterval) Times() (time.Time, time.Time, error) {
```

Comment:
What is this used for?  I don't see it referenced in the change (maybe I missed it).

Reply:
Metrics returns a TimeInterval, so this can be used to parse the go times from the response.

Reply:
I wonder if this should be named `Values()` instead.  I feel like `TimeInterval.Values()` and `Timespan.Values()` reads better.  What do you think?

Reply:
fixed in commit 
f54dc06